### DISCUSSION TITLE
Teta supabase

### DISCUSSION THREAD

**Discussion by fioretti78** [supabase/supabase#8787] [General]
good evening , I created a db with supabase then I tried to integrate it into the teta app by inserting url and anon. now in the page I insert supabase builder by inserting the parameters of the db that is the name of the columns and rows and then I have also inserted a listviewbuilder where the listview takes the data from supabase. and this is where the problem arises I do not get any data. it would therefore seem that the db is empty, instead it has 3 columns and 3 rows. thanks for support

**Comment by GaryAustin1** [comment]
Normally this would be caused by having RLS on the table and no policy or not meeting policy.  It could also be caused by not meeting a filter.

**Reply by GaryAustin1** [reply]
You are not providing much information to help you.  In the Supabase dashboard to do you have RLS (Row Level Security) on?  If so will you meet the select policy, or turn it off to test if that is your issue.
I glanced for some info on Teta setup to link to Supabase and don't find anything documented.  Do you know if your connection information is working?  I assume you would get an error if not, but don't know Teta.   

You may need to wait for someone who has some Teta experience.

**Reply by GaryAustin1** [reply]
Did you try with RLS off?
Also please don't keep adding answers with follow up questions.
You can post what ever would help here.  
You mention the word "support" just want to be clear github discussion is not an official "support" channel and many of us here are not employees.

**Comment by fioretti78** [comment]
thanks for replying, so I can solve by looking on the internet for
some solution to manage the filters?


Il Lun 5 Set 2022, 14:34 GaryAustin1 ***@***.***> ha scritto:

**Comment by fioretti78** [comment]
hello thanks for the support, I have activated rls, in supabase I have
checked many times if the connection to the application is successful and
is connected.
unfortunately, however, when I try to retrieve the data using a widget text
in the value dataset section, I type supabase builder but then I can't see
anything else while instead I should see the header of the db columns.
can I send you some photos?

Il Lun 5 Set 2022, 14:58 GaryAustin1 ***@***.***> ha scritto:

**Comment by fioretti78** [comment]
with the word support I mean the help you are giving me. in Italy we
mean support when you receive assistance of any kind. thanks for your
help and time you dedicate to me


Il Lun 5 Set 2022, 22:13 GaryAustin1 ***@***.***> ha scritto:
